src/models/Settings.js

```diff
@@ -52,6 +52,7 @@ function (Okta, Errors, BrowserFeatures) {
       'features.rememberMe': ['boolean', true, true],
       'features.rememberDevice': ['boolean', true, true],
       'features.rememberDeviceAlways': ['boolean', true, false],
+      'features.autoPush': ['boolean', true, false],
```

Comment:
Sorry for the confusion, I meant `this.model.get(`autoPush`)` would be undefined when `features.autoPush` is `false` because the `autoPush` checkbox won't be [initialized] (https://github.com/okta/okta-signin-widget/pull/57/files#diff-8ccd3af6dc30ed000779477ac78c8de4R78).